src/data/graphql/typeDefs.ts

```diff
@@ -18,6 +18,20 @@ export default gql`
     networkId: Int
   }
 
+  type ParsedEvent {
+    name: String
+    topic: String
+    values: String
```

Comment:
I think I need help typing this properly, since `values` will be an object, of an _mostly -_ unknown shape

Reply:
Hmmm... since graphql doesn't really support _unknown schemas_ (to my knowledge, at least), maybe something like an array of tuples (as key-value pairs) would suffice here? Obviously there are some drawbacks here, such as type safety anywhere these values would be used but... better than nothing 🤷‍♂️

Reply:
Are we sure that we don't know shape of this object? All possible values of this object aren't define by ColonyNetwork? We can probably type all of those key-values pairs and just make them optional as each event return different values?

Reply:
They're unknown in the sense that to know them all, it would be a lot of work to find out. True they all come from the contracts, but basically, we would need to get through all the events and find out what they're values are, ... which are a lot

Reply:
True, but might be worth it, as that would provide support for discriminated unions via fragments which I think would be really helpful. Similar to the `eventContext` from the server events